Net pol doc (#23)
refractionPOINT/lce_doc: docs/net.md

The underlying technology used for VPN is called [WireGuard](https://www.wireguard.com/). WireGuard is a next-generation VPN technology that is promoted for its simplicity, speed, and security.

Clients are available for [Windows, Android, macOS, Linux, iOS and ChromeOS](https://www.wireguard.com/install/). Client configuration is done either through a QR code or a simple configuration file.

## Provisioning

### lc-net-install Service

> This LimaCharlie Service called `lc-net-install` currently only support Windows. It allows you to easily provision

Reviewer: ```suggestion
This LimaCharlie Service called `lc-net-install` currently only supports Windows. It allows you to easily provision
```

> and upgrade a lc-agent to lc-net.

Reviewer: ```suggestion
and upgrade an lc-agent to lc-net.
```

Since it is a LimaCharlie Service it means you can either directly interact with the service or you can issue
service requests via the API and even D&R rules.

To use the service, you will need:

1. The Net version of the Installation Key you want your new lc-net sensors to use: go to the Installation Key section, if you don't have a key create one, and then click the copy-to-clipboard button for the Net key.
2. The Sensor ID (sid) of the sensor you wish to upgrade. If you are using the Service interactively through the web
interface, you may use the Hostname of the sensor and the web interface will auto-complete the Sensor ID.

These steps assume you are using the web interface, but the same basic steps and parameters apply to other methods.

1. Go in the main page of the Organization where you want to upgrade sensors to lc-net.
1. From the left menu, go to the Add-ons section, Services tab, `lc-net-install` service and set to "on".

> 1. From the left meny, go to the Service Request section.

Reviewer: ```suggestion
1. From the left menu, go to the Service Request section.
```

1. Select the `lc-net-install` service from the "Service" drop down.

> 1. In the "iid" section, enter the Installation obtained earlier.

Reviewer: ```suggestion
1. In the "iid" section, enter the Net Installation Key (obtained earlier).
```

  --policy POLICY       literal policy content in JSON or YAML format
```

Policies often contain some generic components:

* `bpf_filter`: this is a [tcpdump-like BPF filter syntax](https://biot.com/capstats/bpf.html) describing matching packets. An empty `bpf_filter` will match all traffic.

### Troubleshooting

> It is sometimes useful to see the list of policies that currentlyapply to a given sensor.

Reviewer: ```suggestion
It is sometimes useful to see the list of policies that currently apply to a given sensor.
```

  ],
  "with_subdomains": false
}
```

### Examples

#### Prevent Use of a Service

> Let's say we want to prevent users on mobiles from accessing Dropbox. Assuming mobile users are tagged as `mobile` as it

Reviewer: ```suggestion
Let's say we want to prevent users on mobile devices from accessing Dropbox. Assuming mobile users are tagged as `mobile` as it
```

can be done either manually, through a D&R rule or through the tags of an Installation Key.

```
{
  "domain": "dropbox.com",
  "tag": "mobile",
  "to_a": [
    "127.0.0.1"
  ],
  "with_subdomains": true
}
```

#### Prevent SSH

> Let's say we want to prevent users in finance from using SSH. Assuming finance users are tagged as `finance` as it

Reviewer: ```suggestion
Let's say we want to prevent users working in the Finance Department from using SSH. Assuming Finance users are tagged as `finance` as it
```
